docs/basics.adoc

```diff
@@ -504,6 +504,38 @@ Connecting to Gremlin-Server (127.0.0.1:8182)... OK (connected to 127.0.0.1:8182
 Run gremlin.sh to connect.
 ----
 
+==== Running JanusGraph Server as a daemon
+
+The JanusGraph https://github.com/JanusGraph/janusgraph/releases[release] also comes with an init script that can be found at `bin/gremlin-server.sh`. The `gremlin-server.sh` script was upgraded to an init script which supports start, stop, status. Its usage is backwards compatible and can still be run in the foreground by invoking `./bin/gremlin-server.sh` with an option parameter of a gremlin server yaml file.
```

Comment:
Thanks for the link. Since markdown GitHub has the same behavior of not displaying the line breaks I'll try to follow this going forward. Looks like I learned something else today.